specification.md

```diff
@@ -2571,6 +2572,31 @@ peer and adjust their behavior appropriately. How exactly the behavior is
 adjusted for future capabilities MUST be defined in the future specification of
 the new capabilities.
 
+### Protobuf Schema Stability
+
+The specification provides the follow stability guarantees of the
+[Protobuf definitions](proto/opamp.proto) for OpAMP 1.0:
+
+- Field types, numbers and names will not change.
+- Names of messages and enums will not change.
+- Numbers assigned to enum choices will not change.
+- Names of enum choices will not change.
+- The location of messages and enums, i.e. whether they are declared at the top lexical
+  scope or nested inside another message will not change.
+- Package names and directory structure will not change.
+- `optional` and `repeated` declarators of existing fields will not change.
+- No existing symbol will be deleted.
+
+Future versions of the OpAMP specification may be extended by modifying the
+Protobuf schema defined in this specification version. The following Protobuf schema
```

Comment:
should we specify how deprecations will be handled for the spec?

Reply:
I haven't put anything thought into the subject of deprecations. I also don't see anything in OTLP spec to use as an inspiration. It is probably worth thinking about, but probably should be a separate PR.